lib/iris/tests/unit/experimental/ugrid/mesh/test_Mesh.py

```diff
@@ -248,6 +252,7 @@ def test_coords(self):
             {"long_name": "foo"},
             {"var_name": "foo"},
             {"attributes": {"test": 2}},
+            {"location": "face"},
```

Comment:
?

Reply:
As above, the addition of this argument expands the coverage of this test to check that the location arument _excludes_ appropriate coords.

Reply:
Agreed offline:

```suggestion
            {"location": "edge"},
```